JSDemos/Demos/TabPanel/Overview/jQuery/index.js

```diff
@@ -1,47 +1,81 @@
 $(() => {
-  DevExpress.setTemplateEngine({
-    compile: (element) => $(element).html(),
-    render: (template, data) => Mustache.render(template, data),
-  });
+  const taskItem = (task) => {
+    const className = `task-item task-item-priority-${task.priority}`;
 
-  const tabPanel = $('#tabpanel-container').dxTabPanel({
-    height: 260,
-    dataSource: tabPanelItems,
-    selectedIndex: 0,
-    loop: false,
+    return `
+      <div class="${className}">
+        <p class="task-item-text">
+          ${task.text}
+        </p>
+
+        <p class="task-item-info">
+          ${task.date} by ${task.assignedBy}
+        </p>
+
+        <div class="task-item-pseudo-button-wrapper">
```

Comment:
I suggest we can use one 'div' element with correspond classes without nested 'i' element to show overflow icon.

Reply:
[01d11075d9f7598a273c29bf190f2bdc1fc3734e]